run/grpc-ping/pkg/api/v1/message.pb.go

```diff
@@ -6,12 +6,13 @@ package ping
 import (
 	context "context"
 	fmt "fmt"
+	math "math"
```

Comment:
Should pb.go files be ignored?

Reply:
I'd prefer to just run on everything, and we don't have many generated pb.go files. So, I'm inclined to keep it for now. Can revert and update the testing to ignore these later, if needed.